### PR TITLE
Added directory "aws-blog-athena-importing-hive-metastores"

### PR DESCRIPTION
**Pull request by nmukerje**
Added directory "aws-blog-athena-importing-hive-metastores" for new blog post.

### CODE REVIEW

File: aws-blog-athena-importing-hive-metastores/exportdatabase.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+p = os.popen(command,"r")
+while 1:
+    line = p.readline()
+    if not line: break
+    tables.append(line)
+
+print "\nFound "+str(len(tables))+" tables in database..."
+
+with open(schema+'_tables.hql',mode='w+') as output:
+   for i in tables:
```
Comment (mbeitchman) [inline aws-blog-athena-importing-hive-metastores/exportdatabase.py]: nit: for t in tables OR for table in tables is more readable

File: aws-blog-athena-importing-hive-metastores/exportdatabase.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+command="hive --database "+schema+" -e 'show tables' -S"
+tables=[]
+separator='!echo \;;'
+
+os.system("rm "+schema+"*.hql 2> /dev/null")
+os.system("rm "+schema+"output 2> /dev/null")
+
+p = os.popen(command,"r")
+while 1:
```
Comment (mbeitchman) [inline aws-blog-athena-importing-hive-metastores/exportdatabase.py]: there's a cleaner way to do this without the risk of the infinite loop. 

sorry if syntax is a little off but you will see the idea

while line = p.readline():
     tables.append(line)

OR

line = p.readline()
while line:
    append
    line = p.readline()

File: aws-blog-athena-importing-hive-metastores/exportdatabase.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    line = p.readline()
+    if not line: break
+    tables.append(line)
+
+print "\nFound "+str(len(tables))+" tables in database..."
+
+with open(schema+'_tables.hql',mode='w+') as output:
```
Comment (mbeitchman) [inline aws-blog-athena-importing-hive-metastores/exportdatabase.py]: '_tables.hql' a constant

File: aws-blog-athena-importing-hive-metastores/executescript.gvy [hunk 4]
```diff
@@ -0,0 +1,42 @@
+import groovy.sql.Sql
+
+def dbDriver = 'com.amazonaws.athena.jdbc.AthenaDriver'
+def dbUrl = 'jdbc:awsathena://athena.us-east-1.amazonaws.com:443/awsdatacatalog/'
+
+java.util.Properties props = new Properties();
+
+//script parameters *** Modify as needed ***
+props.put("s3_staging_dir", "s3://ws-athena-query-results-<<account number>>/staging/");
```
Comment (mbeitchman) [inline aws-blog-athena-importing-hive-metastores/executescript.gvy]: make constants please

File: aws-blog-athena-importing-hive-metastores/executescript.gvy [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+if( !targetfile.exists() ) {
+  println "Error: File does not exist"
+  System.exit(0)
+}
+dataList = targetfile.text.split( ';' )
+print "\nFound "+(dataList.size()-1)+" statements in script...\n"
+sql=Sql.newInstance( dbUrl+args[0], props, dbDriver)
+dataList.eachWithIndex {elem, i ->
+   if (elem.trim()!=";" && elem.trim().size()>0){
+      println "\n"+(i+1)+". Executing :"+elem
```
Comment (mbeitchman) [inline aws-blog-athena-importing-hive-metastores/executescript.gvy]: nit spaceing

File: aws-blog-athena-importing-hive-metastores/executescript.gvy [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+         //execute sql statement
+         row = sql.firstRow(elem)
+         if (row==null) {
+            print "\nresult : OK\n\n"
+         } else {
+            print "\nresult : " +row+"\n\n"
+         }
+      }
+      catch(Exception ex) {
+         println("Caught exception: "+ex);
```
Comment (mbeitchman) [inline aws-blog-athena-importing-hive-metastores/executescript.gvy]: nit: spacing

File: aws-blog-athena-importing-hive-metastores/executescript.gvy (hunk 5 above)
Comment (nmukerje) [inline aws-blog-athena-importing-hive-metastores/executescript.gvy]: Fixed in latest version: Space add before opening curly braces.

Comment (nmukerje) [inline aws-blog-athena-importing-hive-metastores/executescript.gvy]: Fixed in latest version: Space add before opening curly braces.

File: aws-blog-athena-importing-hive-metastores/executescript.gvy (hunk 6 above)
Comment (nmukerje) [inline aws-blog-athena-importing-hive-metastores/executescript.gvy]: Fixed in latest version: Space add before opening curly braces.

File: aws-blog-athena-importing-hive-metastores/exportdatabase.py (hunk 2 above)
Comment (nmukerje) [inline aws-blog-athena-importing-hive-metastores/exportdatabase.py]: Implemented.

while line = p.readline():
   tables.append(line)

File: aws-blog-athena-importing-hive-metastores/exportdatabase.py (hunk 3 above)
Comment (nmukerje) [inline aws-blog-athena-importing-hive-metastores/exportdatabase.py]: Constant added.

File: aws-blog-athena-importing-hive-metastores/exportdatabase.py (hunk 1 above)
Comment (nmukerje) [inline aws-blog-athena-importing-hive-metastores/exportdatabase.py]: Done.

File: aws-blog-athena-importing-hive-metastores/executescript.gvy (hunk 4 above)
Comment (nmukerje) [inline aws-blog-athena-importing-hive-metastores/executescript.gvy]: Constants added.

**Pull request merged**